js/search.js

```diff
@@ -155,7 +155,7 @@
 		var queryAlias;
 		if (query in BattleAliases) {
 			queryAlias = toId(BattleAliases[query]);
-			if (query === 'pika' || query === 'bee' || query === 'sab') searchPasses.shift();
+			if (queryAlias.search(query) === 0) searchPasses.shift();
```

Comment:
Also, `searchPasses.shift()` removes the non-alias search. This makes it impossible to find "Sablenite" on `sab`. We should probably do it the other way around.